### PR TITLE
Test against latest Ruby version and cache bundler

### PR DESCRIPTION
**Pull request by dlackty**

### CODE REVIEW

**Comment by gnowxilef** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: ruby
+sudo: false
+cache: bundler
 rvm:
-  - 2.2.0
+  - 2.2.3
   - 2.1.5
```
Comment (orien) [inline .travis.yml]: Let's update the 2.1 branch also. Ruby 2.1.7 is the latest.

**Comment by orien** [comment]
It looks like the bundler caching has introduced a `vendor` directory that Rubocop is policing. #43 should fix that up.

File: .travis.yml (hunk 1 above)
Comment (dlackty) [inline .travis.yml]: @orien Updated!

**Comment by orien** [comment]
Care to try again?

**Comment by dlackty** [comment]
@orien No problem. Working on.

**Comment by dlackty** [comment]
Done. Also added the latest 2.3.0 as well.

**Comment by orien** [comment]
Thank you!

**Pull request merged**